### DISCUSSION TITLE
Flume Water API (JTW)

### DISCUSSION THREAD

**Discussion by thegizzardwizard** [grafana/grafana-infinity-datasource#563] [Specific APIs]
Hi - Im having a hard time accessing Flume Water API. It has a 2 stage flow and Im not sure how to set it up. It needs to decode a user id from the JWT token. I've included all of their required body params but after setup any queries result in a "oauth2: server response missing access_token"

API Guide is here: https://flumetech.readme.io/reference/get-tokens-1

Thanks.

**Comment by yesoreyeram** [comment]
Grant type password is not supported in the plugin. Only client credentials grant type and jwt grant type are supported.
